assets/javascripts/running.js

```diff
@@ -262,6 +262,21 @@ function initLivelog() {
 
 /********* LIVE LOG END *********/
 
+/********* LIVE TERMINAL *********/
```

Comment:
I don't understand what these comments blocks should be for. The function name is already explicit enough, isn't it?